### DISCUSSION TITLE
Recommended colorizer for LazyVim?

### DISCUSSION THREAD

**Discussion by limaceous-bushwhacker** [LazyVim/LazyVim#2374] [Q&A] [answered]
Title. What plugin the community uses/recommends to preview colors while editing source code?

**Comment by limaceous-bushwhacker** [comment]
Spoke too soon. Came across `NvChad/nvim-colorizer.lua` and it works well. Use:

```
return {
  {
    "NvChad/nvim-colorizer.lua",
    opts = {
      user_default_options = {
        mode = "background",
        names = false,
      },
    },
  },
}
```

**Reply by jyuan0** [reply, marked as answer]
There's also a LazyVim extra called `util.mini-hipatterns` for the plugin `mini.hipatterns`.

**Reply by limaceous-bushwhacker** [reply]
Ty! That works as well.

EDIT: https://www.lazyvim.org/extras/util/mini-hipatterns
